ocs_ci/ocs/constants.py

```diff
@@ -113,12 +113,17 @@
 
 UPI_INSTALL_SCRIPT = "upi_on_aws-install.sh"
 
-DEFAULT_SECRET = 'rook-ceph-csi'
-DEFAULT_BLOCKPOOL = 'rbd'
+DEFAULT_CLUSTERNAME = 'ocs-storagecluster'
+DEFAULT_SECRET = 'rook-csi-rbd-node'
```

Comment:
Hi Vasu, hope I am understanding the change correctly. But if yes, the default secret is not **rook-csi-rbd-node**. There are 2 different secrets now , one for node and one for provisioner. So, we have 4 Secrets :

Also, if we are planning to use the default SCs for all tests, then do we even need to have  a check for secrets (except when Test case itself is about SC and secrets) ? 

RBD
-------
  csi.storage.k8s.io/node-stage-secret-name: **rook-csi-rbd-node**
  csi.storage.k8s.io/node-stage-secret-namespace: openshift-storage
  csi.storage.k8s.io/provisioner-secret-name: **rook-csi-rbd-provisioner**
  csi.storage.k8s.io/provisioner-secret-namespace: openshift-storage

CephFS
-----------
  csi.storage.k8s.io/node-stage-secret-name: **rook-csi-cephfs-node**
  csi.storage.k8s.io/node-stage-secret-namespace: openshift-storage
  csi.storage.k8s.io/provisioner-secret-name: **rook-csi-cephfs-provisioner**
  csi.storage.k8s.io/provisioner-secret-namespace: openshift-storage

Reply:
@nehaberry I think the default secret is not of much use now, There is one test that tries to use default secret and we have to remove it. Since we are going to deal only with default StorageClasses and I wasn't sure where this was used(the old code), I just changed the name to reflect RBD secret.